components/AnimatedImage/library/src/desktopMain/kotlin/org/jetbrains/compose/animatedimage/LocalLoader.kt

```diff
@@ -0,0 +1,21 @@
+package org.jetbrains.compose.animatedimage
+
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import java.io.FileInputStream
+
+class LocalLoader(private val imageUrl: String) : AnimatedImageLoader {
```

Comment:
If we make `suspend load*` function as described earlier, we can make LocalLoader and ResourceLoader internal. It slightly less verbose (`val image = loadExternalAnimatedImage("./image.png")` instead of `val image = FileAnimatedImageLoader("./image.png").load()`).

Otherwise, we need to rename it to `FileAnimatedImageLoader` (because it should contain info about AnimatedImage and info about loading from File), and do the same for other loaders.